src/04-modality-specific-files/03-electroencephalography.md

```diff
@@ -419,9 +423,13 @@ sub-<label>/
 ```
 
 Photos of the anatomical landmarks and/or fiducials are OPTIONAL.
-Please note that the photos may need to be cropped or
-blurred to conceal identifying features prior to sharing, depending on the
-terms of the consent form signed by the participant.
+Please note that the photos may need to be cropped or blurred to conceal
+identifying features prior to sharing, depending on the terms of the consent
```

Comment:
... depending on the terms of the consent THAT IS GIVEN by the participant. 

(It is not about the form, it is about the actual consent, which might also be verbal)